platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/multi_project_configuration_and_execution.adoc

```diff
@@ -13,6 +13,30 @@
 // limitations under the License.
 
 [[configuration_and_execution]]
-= Configuration and Execution time
+= Configuration On Demand
 
-This page has been removed.
+Configuration-on-demand attempts to configure only relevant projects for requested tasks, i.e., it only executes the build script file of projects participating in the build.
```

Comment:
Wondering about the wording here - instead of "executes" is it more correct to use "evaluates"?

Reply:
Thank you for the suggested. I updated it accordingly.